src/components/AddItem.js

```diff
@@ -3,9 +3,10 @@ import { useState } from 'react';
 import { db } from '../lib/firebase.js';
 import useFirebaseSnapshot from '../hooks/useFirebaseSnapshot.js';
 import cleanData from '../utils/cleanData.js';
+// import {navigate} from './'
```

Comment:
🎨 Let's delete this line instead of leaving it commented out

Reply:
done!